src/main/app/src/app/plan-items/human-task-do/human-task-do.component.ts

```diff
@@ -37,6 +37,7 @@ export class HumanTaskDoComponent implements OnInit {
 
   protected form = this.formBuilder.group({});
   protected formFields: FormField[] = [];
+  protected loading = false;
```

Comment:
should booleans start with `is`? So here it would be 
```suggestion
  protected isLoading = false;
```